upsun/10-sulu-upsun.yaml

```diff
@@ -0,0 +1,206 @@
+requirements:
+  - type: file_exists
+    value: composer.json
+  - type: has_composer_package
+    value: sulu/sulu
+
+template: |
+    {{- $varnishServiceName := "" -}}
+    {{- $hasRedisSession := false -}}
+    {{- range $service := $.Services -}}
+        {{- if eq $service.Type "varnish" -}}
+            {{- $varnishServiceName = $service.Name -}}
+        {{- end -}}
+        {{- if eq $service.Name "redissession" -}}
+            {{- $hasRedisSession = true -}}
+        {{- end -}}
+    {{- end -}}
+    routes:
+        {{ if $varnishServiceName -}}
+        "https://{all}/admin": { type: upstream, upstream: "{{.Slug}}:http", cache: { enabled: false } }
+        "https://{all}/": { type: upstream, upstream: "{{ $varnishServiceName }}:http", cache: { enabled: false } }
+        {{- else -}}
+        "https://{all}/": { type: upstream, upstream: "{{.Slug}}:http", cache: { enabled: false } }
+        {{- end }}
+        "http://{all}/": { type: redirect, to: "https://{all}/" }
+
+    services:{{ if not $.Services }} {}{{ end }}
+        {{- range $service := $.Services }}
+        {{ $service.Name }}:
+            type: {{ $service.Type }}{{ if $service.Version }}:{{ $service.Version }}{{ end }}
+            {{- if eq $service.Type "varnish" }}
+            relationships:
+                application: "{{$.Slug}}:http"
+            configuration:
+                vcl: !include
+                    type: string
+                    path: varnish.vcl
+            {{- end }}
+        {{- end }}
+
+    applications:
+        {{.Slug}}:
+            source:
+                root: "/"
+
+            type: php:{{.PhpVersion}}
+
+            runtime:
+                extensions:
+                    {{ range $ext := $.PHPExtensions -}}
+                    {{- if php_extension_available $ext $.PhpVersion -}}
+                    - {{ $ext }}
+                    {{ end -}}
+                    {{- end }}
+
+            variables:
+                php:
+                    {{- if php_at_least "7.4" }}
+                    opcache.preload: config/preload.php
+                    {{- end }}
+                    {{- if $hasRedisSession }}
+                    session.save_handler: redis
+                    session.save_path: "tcp://${REDISSESSION_HOST}:${REDISSESSION_PORT}"
```

Comment:
@nicolas-grekas While testing a multi-node setup with Redis as session storage, I believe I found an error in the upsun documentation: https://docs.upsun.com/add-services/redis.html#use-redis-as-a-handler-for-php-sessions

Incorrect service name and missing {}:
`$SESSIONSTORAGE_HOSTNAME:$SESSIONSTORAGE_PORT` => `${REDISSESSION_HOST}:${REDISSESSION_PORT}`

I’m happy to create a PR, but I’m not sure where or how the documentation is maintained.

/cc @fabpot